sources/packages/forms/src/form-definitions/supportingusersparent2024-2025.json

```diff
@@ -488,19 +488,19 @@
                   "label": "How much will you and your spouse/common-law partner be giving the student to help meet their specific educational costs during this study period?",
                   "description": "Please include money, total cashed Registered Education Savings Plans (RESPs) and scholarship trust funds",
                   "prefix": "$",
+                  "applyMaskOn": "change",
                   "mask": false,
-                  "spellcheck": true,
                   "tableView": false,
-                  "defaultValue": [
-                    null
-                  ],
                   "currency": "CAD",
                   "inputFormat": "plain",
+                  "truncateMultipleSpaces": false,
                   "validate": {
                     "required": true
                   },
+                  "validateWhenHidden": false,
                   "key": "parentalContributions",
                   "type": "currency",
```

Comment:
I did not follow the change in these components as it is still of type currency. Same in other places.

Reply:
Apologies - I have converted the currency components now.